src/dns/coredns.rs

```diff
@@ -227,6 +227,14 @@ impl CoreDns {
                                 reply(sender, src_address, &req);
                             } else {
                                 debug!("Not found, forwarding dns request for {:?}", name);
+                                match env::var("AARDVARK_NO_PROXY") {
```

Comment:
Can we lookup the env var once and store it as bool instead of looking it up for every single request.

Reply:
fixed.